lesson04/demo.js

```diff
@@ -0,0 +1,9 @@
+function numberToSquare () {
+    let input = prompt ('Enter a number');
+    let numberToSquare = input
+    let result = 0;
+
+    result = numberToSquare * Number(input);
```

Comment:
this is a little over complicated. You don't need the numberToSquare variable since we're just using input twice. Plus you don't apply Number() on line 3, which could lead to bugs.

`result = Number(input) * Number(input)`